examples/nordic/nrf5x/src/i2c_accel.zig

```diff
@@ -57,30 +57,12 @@ pub fn main() !void {
     try dev.setup();
 
     while (true) {
-        //   DELETEME>>
```

Comment:
I have a little vim macro that adds and removes these blocks, the fact that this was committed shows I messed up 🙃 .

Reply:
Whoops! I'll ask before merging next time. I'm away this week but I should be able to test my new ICM-20948 this soon on some different hardware!